src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -228,9 +231,22 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.AddHostingConfigOptions(configuration);
             services.ConfigureOptions<ExtensionRequirementOptionsSetup>();
 
-            // Refresh LanguageWorkerOptions when HostBuiltChangeTokenSource is triggered.
+            // Refresh WorkerConfigurationResolverOptions and LanguageWorkerOptions when HostBuiltChangeTokenSource is triggered.
+            services.ConfigureOptionsWithChangeTokenSource<WorkerConfigurationResolverOptions, WorkerConfigurationResolverOptionsSetup, HostBuiltChangeTokenSource<WorkerConfigurationResolverOptions>>();
```

Comment:
The DI of `WorkerConfigurationResolverOptionsSetup` has been done similar to `LanguageWorkerOptionsSetup` because the ultimate purpose of both is to get WorkerConfigs and `LanguageWorkerOptionsSetup` uses `WorkerConfigResolverOptions` an input.